inspirehep/version.py

```diff
@@ -22,5 +22,27 @@
 
 from __future__ import absolute_import, division, print_function
 
+import logging
+import pkg_resources
 
-__version__ = '0.1.0'
+from git import Repo
+
+logger = logging.getLogger(__name__)
+
+
+
+def _get_git_tag_version():
+    try:
+        git_repo_path = pkg_resources.get_distribution('inspirehep').location
+        repo = Repo(git_repo_path)
+        tags = sorted(repo.tags, key=lambda t: t.commit.committed_datetime)
+        latest_tag = tags[-1]
+        logger.info('Running in git tag: {}'.format(latest_tag))
```

Comment:
maybe put logging outside in `__init__` to be more obvious. Since logging is the whole reason why we're doing this s**t

Reply:
good point 👍